src/api.js

```diff
@@ -111,15 +112,15 @@ var schema = function(validator, options) {
           contentType: (req.headers['content-type'] || null),
         });
       }
-      var errors = validator.check(req.body, options.input);
-      if (errors) {
+      var error = validate(req.body, options.input);
+      if (error) {
         debug("Request payload for %s didn't follow schema %s",
               req.url, options.input);
         return res.reportError(
           'InputValidationError',
           "Request payload must follow the schema: {{schema}}\n" +
-          "Errors: {{errors}}", {
-          errors,
+          "Error: {{error}}", {
```

Comment:
I suspect we don't need the `Error: ` part... is the `error` variable a full error message with everything needed?

So maybe `res.reportError('InputValidationError', error, {schema: options.input, payload: req.body});` is fine...